js/app.js

```diff
@@ -1,17 +1,16 @@
-import React, { Component } from 'react';
+import React from 'react';
+import * as firebase from 'firebase';
 import AppNavigator from './appNavigator';
 
-class App extends Component {
+firebase.initializeApp({
+  apiKey: 'AIzaSyCY5F5QqKqaVTvwvQQf5EQgu2apNIfG1-0',
```

Comment:
Looking further into firebase it seems to be standard practise to expose all those variables.

https://groups.google.com/forum/#!msg/firebase-talk/-u2wBWSETds/MWP5BWtnFwAJ